tests/functional.suite.yml

```diff
@@ -0,0 +1,13 @@
+# Codeception Test Suite Configuration
+#
+# Suite for functional tests
+# Emulate web requests and make WordPress process them
+
+actor: FunctionalTester
+modules:
+    enabled:
+        - WPDb
+        - WPBrowser
+        # - WPFilesystem
```

Comment:
Should this be removed or kept in?

Reply:
It has no value unless it's needed by the tests, and we can't know that unless tests are written, so can remove it for now.

Reply:
Will people have to add these files themselves in the project root tests directory? Or are pointing Codeception to them somehow? Sorry for the all the questions, just wrapping my head around it!

Reply:
We can figure out if we can provide some helpers to generate suites, Codeception and wp-browser has stuff that can be useful, just need to make sure it works with our custom config / locations.